hack/run-locally.sh

```diff
@@ -152,9 +154,20 @@ function create_cluster() {
     trap "trap - TERM && kill -- -$$" INT TERM EXIT
 }
 
+function detect_platform() {
```

Comment:
fwiw if you're only doing this when the cluster is up, you can do `oc get infrastructure cluster -o jsonpath='{.status.platform}'`